legacy/firmware/transaction.c

```diff
@@ -738,7 +740,7 @@ uint32_t tx_serialize_output(TxStruct *tx, const TxOutputBinType *output,
   r += 8;
 #if !BITCOIN_ONLY
   if (tx->is_decred) {
-    uint16_t script_version = output->decred_script_version & 0xFFFF;
+    uint16_t script_version = DECRED_SCRIPT_VERSION & 0xFFFF;
```

Comment:
same. we are leaving in `TxOutputBinType.decred_script_version`, but not using it here.

Reply:
Since the function definition says `TxOutputBinType` this should be used only for BinType, so I am reverting this change.